web_src/js/features/repo-editor.js

```diff
@@ -148,7 +148,7 @@ export function initRepoEditor() {
       dirtyClass: dirtyFileClass,
       fieldSelector: ':input:not(.commit-form-wrapper :input)',
       change() {
-        const dirty = $(this).hasClass(dirtyFileClass);
+        const dirty = this[0]?.classList.contains(dirtyFileClass);
         commitButton.disabled = !dirty;
       },
```

Comment:
Wow, does `areYouSure` really assign `this` to a jQuery collection? That is so not-jquery like.

Reply:
It does appear to be the case:

 https://github.com/codedance/jquery.AreYouSure/blob/60f199ce6354f9eb8b196575a59610a3c3cebd36/jquery.are-you-sure.js#L130

Can't even prefix `this` with `$`, damnit 😆

Reply:
We can avoid `this` by using first argument `$form`.

Reply:
Done in suggestions

Reply:
Tested, it works.